test/IntegrationTests/MSTest.Acceptance.IntegrationTests/NativeAotTests.cs

```diff
@@ -57,6 +42,36 @@ namespace MyTests;
 [TestClass]
 public class UnitTest1
 {
+    [ClassInitialize]
+    public static void ClassInit(TestContext testContext)
+    {
+    }
+
+    [ClassCleanup]
+    public static void ClassClean(TestContext testContext)
```

Comment:
No param for the cleanup methods.

```suggestion
    public static void ClassClean()
```